damnit/gui/plot.py

```diff
@@ -22,22 +22,24 @@
 
 log = logging.getLogger(__name__)
 
-class Canvas(QtWidgets.QDialog):
+CORRELATION_MSG = """\ 
```

Comment:
There's a trailing space here that gives me a syntax warning:
```suggestion
CORRELATION_MSG = """\
```